filters/utilization_threshold.cpp

```diff
@@ -85,6 +85,12 @@ Try<Nothing> UtilizationThresholdFilter::consume(const ResourceUsage& product) {
   this->previousSamples->clear();
   this->previousSamples = std::move(newSamples);
 
+  if (this->previousSamples->empty()) {
+    // Don't log it in real env.
+    LOG(INFO) << name << "No Executor in Usage.";
```

Comment:
Can you add a bit more debug info? INFO is usually prone to be spammed, so it should be actionable